packages/logger/src/register.ts

```diff
@@ -13,7 +13,7 @@ export class LoggerPlugin extends Plugin {
 	 */
 	public static [preGenericsInitialization](this: SapphireClient, options: ClientOptions): void {
 		options.logger ??= {};
-		options.logger.instance = new Logger(options.logger);
+		options.logger.instance = options.logger?.instance ?? new Logger(options.logger);
```

Comment:
I am not a maintainer of Sapphire by any means, but just wanted to share a possible suggestion for using the [nullish coalescing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) operator.
```suggestion
		options.logger.instance ??= new Logger(options.logger);
```